tests/test_exception.py

```diff
@@ -25,3 +27,30 @@ async def coro(val: int) -> None:
         await coro(1)
 
     check_lru(coro, hits=2, misses=2, cache=1, tasks=0)
+
+
+@pytest.mark.skipif(
+    reason="Memory leak is not fixed for PyPy3.9",
```

Comment:
This suggests that it should be fixed at some point, in which case we probably want an xfail instead of a skip.